helmfile/values/service-cluster-networkpolicy.yaml.gotmpl

```
@@ -2,8 +2,20 @@ additional:
 {{- if .Values.networkPolicies.additional }}
   {{ toYaml .Values.networkPolicies.additional | nindent 2 }}
 {{- end }}
+
+global:
+  apiserver:
+    ip: {{- toYaml .Values.networkPolicies.global.apiserver.ip | nindent 6 }}
+    port: {{- toYaml .Values.networkPolicies.global.apiserver.port | nindent 6 }}
```

This doesn't need to be a list right?
```suggestion
    port: {{ .Values.networkPolicies.global.apiserver.port }}
```